allauth/mfa/tests/test_views.py

```diff
@@ -71,7 +73,9 @@ def test_activate_totp_success(
                 {
                     "code": "123",
                 },
+                **{"HTTP_USER_AGENT": "test"},
             )
+    assert len(mail.outbox) == 1
```

Comment:
Can you switch to the `pytest` `mailoutbox` fixture?:

https://pytest-django.readthedocs.io/en/latest/helpers.html#mailoutbox

Reply:
switched to pytest mailoutbox everywhere.